crates/opencascade-sys/build.rs

```diff
@@ -15,6 +15,7 @@ fn main() {
         .build();
 
     println!("cargo:rustc-link-search=native={}", dst.join("lib").display());
+    println!("cargo:rustc-link-search=native={}", dst.join("win64/gcc/lib").display());
```

Comment:
Do you think you could print out the rough directory structure in `dst` for what a windows build gives? I don't have a windows machine handy at the moment.

Reply:
You end up with roughly this: 
![image](https://user-images.githubusercontent.com/3913937/201444327-1cf74265-fda1-4b58-b046-d616d07356be.png)

Reply:
Thank you, this is helpful!

Reply:
I just checked on a windows machine with the native toolchain `x86_64-pc-windows-msvc`.

The directory structure is `opencascade-sys/out/win64/vc14/lib`

The `vc14` part is a bit unfortunate as it might change with different versions of visual studio, but I'll see what can be done here.

Reply:
I guess we could try to derive it from the `TARGET` environment variable. The trick is that it doesn't match nicely at all. So it would need to be switch cased for all options.

Reply:
Just pushing an update. I dug through the `CMakeLists` and found:
```
# OCCT headers: <prefix>/inc for windows,
#               <prefix>/include/opencascade-7.0.0 for unix
if (NOT DEFINED INSTALL_DIR_INCLUDE)
```
They have both `INSTALL_DIR_LIB` and `INSTALL_DIR_INCLUDE`.
By setting those we can guarantee the output and not have to worry about the VS compiler version.

Reply:
Oh good catch @DSchroer!